src/chaos/controller.go

```diff
@@ -77,27 +77,46 @@ func (cc *ChaosController) deleteNode(namespace string, podName string) error {
 	return cc.DynamicClient.Resource(podGVR).Namespace(namespace).Delete(context.TODO(), podName, v1.DeleteOptions{})
 }
 
-// configConfigPath    - path to k8s cluster config
-// killNodes 	- to kill or not to kill affected nodes after each run
-func RunChaosExperiment(namespace string, chaosConfigPath string) error {
+func RunChaosExperiment(namespace string, manifestObj *unstructured.Unstructured) error {
```

Comment:
You don't need Obj postfix. Golang is a typed language, it's clear here that it's not a string.